.circleci/run_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+engine=$1
+testCommand=$2
+branch=`git rev-parse --abbrev-ref HEAD`
+echo $branch
+modifiedFiles="git diff --name-only $branch master"
+
+git checkout master
+git reset --hard origin/master
+git checkout -
```

Comment:
Would it be possible to you use `origin/master` instead of `master` in the subsequent `git diff`'s and thus avoid this?

Reply:
Let me try! I had to use all this because it seems that, when pulling, CircleCI only pulls the branch it's building, so when I compare to `master` I had no changes because git was creating `master` from the current branch. That's why I wrote this piece of code. Let me see if your solution works :)